examples/biology_qa.py

```diff
@@ -19,17 +19,23 @@
 @task
 def biology_qa() -> Task:
     return Task(
+        model="openai/o4-mini",
         dataset=example_dataset(
             name="biology_qa",
             sample_fields=FieldSpec(input="question", target="answer"),
         ),
-        solver=[
-            use_tools(
+        sandbox=("docker", "./intervention/multi_tool/compose.yaml"),
+        solver=react(
+            tools=[
+                bash(),
                 web_search(
-                    providers={"openai": openai_options, "tavily": tavily_options},
-                )
-            ),
-            generate(),
-        ],
+                    providers={
+                        "openai": openai_options,
+                        "anthropic": None,
```

Comment:
I think maybe this should be `True` instead of `None` ? (`None` reads too much like "do not use anthropic)

Reply:
Done. We now document and type for `True` in that spot and accept `None` at runtime for backward compatibility.